.github/workflows/_swift.yml

```diff
@@ -5,172 +5,79 @@ on:
 
 jobs:
   build:
-    name: build-${{ matrix.sdk }}
-    runs-on: ${{ matrix.runs-on }}
+    name: ${{ matrix.job_name }}
+    runs-on: macos-15
+    permissions:
+      contents: read
+      id-token: "write"
+    env:
+      XCODE_VERSION: "16.2"
     strategy:
       fail-fast: false
       matrix:
         include:
-          - sdk: macosx
-            runs-on: macos-14
-            platform: macOS
-            xcode: "15.2"
-            destination: platform=macOS
-          - sdk: iphoneos
-            runs-on: macos-14
-            platform: iOS
-            xcode: "15.2"
-            destination: generic/platform=iOS
-    permissions:
-      contents: read
-      id-token: "write"
-    defaults:
-      run:
-        working-directory: ./swift/apple
+          - job_name: build-ios
+            rust-targets: aarch64-apple-ios
+            build-script: scripts/build/ios-appstore.sh
+            ARTIFACT_PATH: "./Firezone.ipa"
```

Comment:
So all env variables here are by convention upper-cased? Don't feel strongly about it but the convention seems a bit unnecessary.

Also would prefer all "scripts" to be grouped together.

Reply:
👍